docs/DE/hilfe/community.md

```diff
@@ -15,7 +15,7 @@ Die Looper-Community tauscht sich intensiv über (geschlossene) Facebook-Gruppen
 In Github gibt es die Möglichkeit, direkten Kontakt zu Entwicklern aufzunehmen, Bugs/Issues zu melden und sich mit anderen aus der Community auszutauschen (grundsätzlich in Englisch):
 
 * **AndroidAPS (Milos Kozak)**: [https://github.com/MilosKozak/AndroidAPS](https://github.com/MilosKozak/AndroidAPS)
-* **AndroidAPS Dokumentation**: [https://github.com/openaps/AndroidAPSdocs](https://github.com/openaps/AndroidAPSdocs)
```

Comment:
This is a change to the old wiki that is only there until all parts are migrated.